### DISCUSSION TITLE
LTE Module

### DISCUSSION THREAD

**Discussion by ManofNorth** [danielewood/sierra-wireless-modems#151] [General]
Do the Lynq L501 and the MC7455 LTE modules have the same function for creating a wireless RPi router? 

Do they allow for users to connect wirelessly?

**Comment by ManofNorth** [comment]
I have a Windows 10 laptop, for your "EM7565/EM7455/MC7455 - Modem Configuration" will this setup not work? 
I have an RPi 4B, what else do I need besides ROOter?
My goal is to have the RPi as a stand-alone router.

**Comment by syspringdawn** [comment]
I have an EM7455 in a sled connected to an RPi4B running ROOter. I use a separate AP as the wifi in the Pi isnt great and will only run one interface at a time, either 2.4 or 5. Works very well as mobile broadband.

On 22/08/2023 14:25, ManofNorth wrote:

I have a Windows 10 laptop, for your "EM7565/EM7455/MC7455 - Modem Configuration" will this setup not work?
I have an RPi 4B, what else do I need besides ROOter?
My goal is to have the RPi as a stand-alone router.
